examples/test.cpp

```diff
@@ -0,0 +1,6 @@
+#include "../include/featurelib.h"
+
+int main() {
+featurelib instance();
```

Comment:
Fix this line's indentation. Also, CLion warns me about these parantheses and I think it's right, so remove them.

Reply:
done